packages/vsphere-cpi/charts/templates/vsphere-cpi-ds.yaml

```diff
@@ -0,0 +1,74 @@
+# Source: https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/v1.19/vsphere-cloud-controller-manager.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  labels:
+    component: cloud-controller-manager
+    tier: control-plane
+    k8s-app: vsphere-cloud-controller-manager
+  name: vsphere-cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      k8s-app: vsphere-cloud-controller-manager
```

Comment:
Technically we should be limiting this to linux only with tolerations, but if it is local cluster only then we could skip that part.

Reply:
We don't care about windows. Chart.yaml annotations specifically call this out as a linux only chart